src/ConcreteOperations/minkowski_sum.jl

```diff
@@ -243,6 +243,32 @@ function minkowski_sum(Z1::AbstractZonotope{N}, Z2::AbstractZonotope{N}) where {
     return Zonotope(cnew, Gnew)
 end
 
+"""
+    minkowski_sum(X::AbstractSingleton, Y::AbstractSingleton)
+
+Concrete Minkowski sum of a pair of singletons.
+
+### Input
+
+- `X` -- singleton
+- `Y` -- singleton
+
+### Output
+
+A singleton
+
+### Algorithm
+
+The singleton obtained by summing the elements in `X` and `Y`.
+"""
+function minkowski_sum(X::AbstractSingleton, Y::AbstractSingleton)
+    return Singleton(element(X) + element(Y))
+end
+
+minkowski_sum(X::AbstractSingleton, Y::ZeroSet) = X
+minkowski_sum(X::ZeroSet, Y::AbstractSingleton) = Y
+minkowski_sum(X::ZeroSet, Y::ZeroSet) = X
```

Comment:
I wonder if we should add dimension checks.

Reply:
ok :+1:  as we usually we with `@assert` in this library